### ISSUE TITLE
Document that disabling `enable-cadvisor-json-endpoints` flag does not disable stats summary endpoint

### ISSUE THREAD

**Issue by adammw** [kubernetes/kubernetes#96483]
**What would you like to be added**:
Update the help text for kubelet's `enable-cadvisor-json-endpoints` flag to explicitly mention that it has no effect over the `/stats/summary` endpoint.

**Why is this needed**:
The help text for kubelet's `enable-cadvisor-json-endpoints` flag describes it as "Enable cAdvisor json `/spec` and `/stats/*` endpoints." which would seem to include `/stats/summary`, however this is not the case as that endpoint is required by kube-metrics-server and exposes stats from cadvisor or CRI (depending on the stat provider selected internally).

**Related Issues**:
https://github.com/kubernetes/kubernetes/issues/68522

/sig node
/sig instrumentation
/kind documentation

**Comment by joadavis** [comment]
/assign

This should be updated in
https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/kubelet.md
as well as 
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L372
(unless there is some automated process that updates the website from the options in code that I'm not aware of).

**Comment by joadavis** [comment]
To summarize a conversation in slack, it appears there is a documented procedure for updating the website found at https://kubernetes.io/docs/contribute/generate-ref-docs/kubectl/
The procedure is long, so it is tempting to just copy in the small amount of text that was changed from options.go into a website PR rather than pulling down so many moving parts.

In any case, we likely should keep this issue open until both the change to options.go and an associated website change are merged.

**Comment by ehashman** [comment]
@joadavis can you link to the Slack convo?

**Comment by joadavis** [comment]
Can do - https://kubernetes.slack.com/archives/C20HH14P7/p1606936008188200

The conversation ended with a question whether there is a bug that would prevent an automated update of the documentation or if we need to submit a PR now that the fix has merged.  I searched for an issue but didn't find a match. So we can either try to find someone on sig-docs who might know if there is an issue if it should be updated automagically, or just fix the doc if the website didn't get updated (and I just checked the master branch and it has not been updated yet).
